spec/integration/test_runner_spec.rb

```diff
@@ -54,6 +54,7 @@ def local_spreadsheet_needs_replacing?(local, remote)
     end
   end
 
+  # rubocop:disable Lint/ConstantDefinitionInBlock
```

Comment:
Rather than disabling the cop, could we just move these constants outside the block (say to line 3)?